01.fizzbuzz/fizzbuzz.rb

```diff
@@ -0,0 +1,18 @@
+numbers = [*1..20]
+
+numbers.each do |number|
```

Comment:
変数に入れず、これだけでも十分だと思いますー。

```suggestion
[*1..20].each do |number|
```

ただ、配列にせず、範囲（Range）をそのままループさせる方が若干シンプルです。

```suggestion
(1..20).each do |number|
```

こんな書き方もありますね。


```suggestion
1.upto(20) do |number|
```

お好きな書き方を使ってくださいー。